packages/@aws-cdk/aws-dynamodb-global/lib/aws-dynamodb-global.d.ts

```diff
@@ -0,0 +1,98 @@
+import dynamodb = require("@aws-cdk/aws-dynamodb");
+import cdk = require("@aws-cdk/cdk");
+import { LambdaGlobalDynamoDBMaker } from "./lambda-global-dynamodb";
+import { MultiDynamoDBStack } from "./multi-dynamodb-stack";
+/**
+ * NOTE: These props should match dynamodb.TableProps exactly
+ * EXCEPT for tableName is now required (for global tables to work, the
+ * table name must match across regions)
```

Comment:
merge issue?

Reply:
Well apparently that package did not have a `.gitignore` file...

Reply:
Ah no, looks like I took on hitchhikers as I passed through a branch.